recipes/server_setup.rb

```diff
@@ -143,6 +143,14 @@
         end
       end
 
+      #Ensure that all peers are actually in the correct state before we try to make the volume
+      peers.each do |peer|
+        bash "reprobe-peer" do
+          command "gluster peer detach #{peer}; gluster peer probe #{peer}"
+          only_if %Q{"execute "gluster peer status | grep -A 2 #{peer} | tail -1 | grep 'Peer in Cluster (Connected)'"}
```

Comment:
Woops, yes, you're right. Let me double check this. Currently having firewall issues which are blocking me deploying the cluster. Will report back with the Chef::Log add as well once that's resolved and tested (need to somehow kill a peer halfway through the run to test it)